util/bloom.cc

```diff
@@ -263,54 +249,6 @@ void FullFilterBitsReader::GetFilterMeta(const Slice& filter,
   *num_lines = DecodeFixed32(filter.data() + len - 4);
 }
 
-void FullFilterBitsReader::FilterPrepare(const uint32_t& hash,
-                                         const Slice& filter,
-                                         const uint32_t& num_lines,
-                                         uint32_t* bit_offset) {
-  uint32_t len = static_cast<uint32_t>(filter.size());
-  if (len <= 5) return;  // remain the same with original filter
```

Comment:
Is this checking gone?

Reply:
It's checked on line 195 (was 193), in (what used to be) the caller. Therefore, this precondition check was arguably redundant, being in a private function. And now that private function is gone. :)

Reply:
OK then.